contracts/ChannelManager.sol

```diff
@@ -210,10 +210,10 @@ contract ChannelManager {
         ));
  
         // TODO show arguments / figure out what's going on 
-        // bytes32 signTypedDataFingerprint = keccak256(abi.encodePacked(
-        //     keccak256(abi.encodePacked("bytes32 hash")),
-        //     keccak256(abi.encodePacked(fingerprint))
-        // ));
+        bytes32 signTypedDataFingerprint = keccak256(abi.encodePacked(
+             keccak256(abi.encodePacked("bytes32 hash")),
```

Comment:
TBH is there any special meaning of this? Isn't that just a string "bytes32 hash" thats hashed, and then ABI encoded with a fingerprint again? Wouldn't it be simplier if both parties just sign a fingerprint the same way?

Reply:
@drozdziak1 the outer hashing should be there as EC signs hashes (or, lets say, any 32 bytes as keccak256 yields 32 bytes). Double hashing occurs later with `fingerprint` - it could be simplified as keccak256(keccak256(fingerprint)).

Reply:
This is some kind of metamask thing